### ISSUE TITLE
Install script broken due to incorrect sbom checksum comparison

### ISSUE THREAD

**Issue by aThorp96** [golangci/golangci-lint#6728]
### Welcome

- [x] Yes, I'm using a binary release within 2 latest [releases](https://github.com/golangci/golangci-lint/releases). Only such installations are supported.
- [x] Yes, I've searched similar [issues on GitHub](https://github.com/golangci/golangci-lint/issues) and didn't find any.
- [x] Yes, I've read the `typecheck` section of the [FAQ](https://golangci-lint.run/docs/welcome/faq/#why-do-you-have-typecheck-errors).
- [x] Yes, I've tried with the standalone [linter](https://golangci-lint.run/docs/linters/) if available (e.g., gocritic, go vet, etc.).
- [x] I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)

### How did you install golangci-lint?

Official binary

### Description of the problem

It appears that when extracting the checksum for a given artifact, the install script's grep extracts both the artifact's hash and the artifact's sbom hash. The artifact is checked against both hashes and of course its hash fails to match the sbom hash

### Version of golangci-lint

NA

### Configuration

<details>

```console
$ grep --version
grep (GNU grep) 3.12
Copyright (C) 2025 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Mike Haertel and others; see
<https://git.savannah.gnu.org/cgit/grep.git/tree/AUTHORS>.

grep -P uses PCRE2 10.47 2025-10-21


$curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b /home/athorp/.local/bin v2.12.2
```

</details>

### Go environment

<details>

```console
 $ go version
go version go1.26.5 linux/amd64
```

</details>

### Verbose output of running

<details>

```console
$ curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh |  sh -s -- -d -b /home/athorp/.local/bin v2.12.2
golangci/golangci-lint info checking GitHub for tag 'v2.12.2'
golangci/golangci-lint debug http_download https://github.com/golangci/golangci-lint/releases/v2.12.2
golangci/golangci-lint info found version: 2.12.2 for v2.12.2/linux/amd64
golangci/golangci-lint debug downloading files into /tmp/tmp.9FbkspvA3r
golangci/golangci-lint debug http_download https://github.com/golangci/golangci-lint/releases/download/v2.12.2/golangci-lint-2.12.2-linux-amd64.tar.gz
golangci/golangci-lint debug http_download https://github.com/golangci/golangci-lint/releases/download/v2.12.2/golangci-lint-2.12.2-checksums.txt
golangci/golangci-lint err hash_sha256_verify checksum for '/tmp/tmp.9FbkspvA3r/golangci-lint-2.12.2-linux-amd64.tar.gz' did not verify 8df580d2670fed8fa984aac0507099af8df275e665215f5c7a2ae3943893a553
fd3a137c7e722128143cc8932bcaa00bc73e10adadee18bdb0893453edb2c137 vs 8df580d2670fed8fa984aac0507099af8df275e665215f5c7a2ae3943893a553
```

</details>

### A minimal reproducible example or link to a public repository

<details>

```go
// NA
```

</details>

### Validation

- [x] Yes, I've included all the information above (example, version, config, etc.).

### Supporter

- [ ] I am a [sponsor/backer](https://donate.golangci.org) of this project.

**Comment by ldez** [comment]
Duplicate of #6539, #6537, #6555, #6556, #6540, #6568, #6570, #6572

---

> [!IMPORTANT] 
> If you are using the install script from the `master` branch:
> `https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh`
> This branch is not used anymore, we are using `main`.
>
> We recommend using `https://golangci-lint.run/install.sh` as URL to install golangci-lint.
> 
> https://golangci-lint.run/docs/welcome/install/local/#binaries

---

**Comment by aThorp96** [comment]
Thanks @ldez. I searched for open issues since I was targeting master, but didn't think to check closed issues. Switching to use `https://golangci-lint.run/install.sh` works 👍🏼
